src/chainlit/message.py

```diff
@@ -153,10 +154,18 @@ def __init__(
         actions: Optional[List[Action]] = None,
         elements: Optional[List[ElementBased]] = None,
     ):
-        self.content = content
+        self.language = language
+
+        if type(content) is dict:
```

Comment:
let's allow subclasses as well

```suggestion
        if isinstance(content, dict):
```